legato-api/src/main/java/org/opendaylight/legato/LegatoCosProfileController.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 2014 Cisco Systems, Inc. and others.  All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ */
+
+
+
+package org.opendaylight.legato;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.opendaylight.controller.md.sal.binding.api.DataBroker;
+import org.opendaylight.controller.md.sal.binding.api.DataTreeIdentifier;
+import org.opendaylight.controller.md.sal.binding.api.DataTreeModification;
+import org.opendaylight.controller.md.sal.binding.api.WriteTransaction;
+import org.opendaylight.controller.md.sal.common.api.data.LogicalDatastoreType;
+import org.opendaylight.legato.util.LegatoConstants;
+import org.opendaylight.legato.util.LegatoUtils;
+import org.opendaylight.unimgr.api.UnimgrDataTreeChangeListener;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.MefGlobal;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.BwpFlowParameterProfiles;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.CosProfiles;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.CosProfilesBuilder;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.cos.profiles.Profile;
+import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.cos.profiles.ProfileKey;
+import org.opendaylight.yangtools.concepts.ListenerRegistration;
+import org.opendaylight.yangtools.yang.binding.InstanceIdentifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Optional;
+//import org.opendaylight.yang.gen.v1.urn.mef.yang.mef.global.rev171215.mef.global.color.mapping.profiles.ProfileKey;;
+public class LegatoCosProfileController extends UnimgrDataTreeChangeListener<Profile>{
+
+	public LegatoCosProfileController(DataBroker dataBroker) {
+		super(dataBroker);
+		registerListener();
+		// TODO Auto-generated constructor stub
+	}
+
+	private static final Logger LOG = LoggerFactory
+			.getLogger(LegatoCosProfileController.class);
+
+
+	private static final InstanceIdentifier<Profile> PROFILE_IID = InstanceIdentifier
+			.builder(MefGlobal.class).child(CosProfiles.class).child(Profile.class).build();
+	
+	
```

Comment:
Have you seen these warnings during compilation?

[WARN] /misc/data/users/mahesh/src/git/unimgr/legato-api/src/main/java/org/opendaylight/legato/LegatoCosProfileController.java:52: 'method def modifier' have incorrect indentation level 8, expected level should be 4. [Indentation]
[WARN] /misc/data/users/mahesh/src/git/unimgr/legato-api/src/main/java/org/opendaylight/legato/LegatoCosProfileController.java:52:1: Line contains a tab character. [FileTabCharacter]
[WARN] /misc/data/users/mahesh/src/git/unimgr/legato-api/src/main/java/org/opendaylight/legato/LegatoCosProfileController.java:53:1: Line contains a tab character. [FileTabCharacter]